.cirrus.yml

```diff
@@ -6,7 +6,7 @@ env:
     #### Global variables used for all tasks
     ####
     # Name of the ultimate destination branch for this CI run, PR or post-merge.
-    DEST_BRANCH: "v2.2"
+    DEST_BRANCH: "v2.2.1-rhel"
```

Comment:
The other PR (#9324) did not change this line. Are you fixing an oversight in the setup of this `-rhel` branch? If so, could you update the commit message accordingly?

Reply:
yes